core/src/main/java/org/infinispan/configuration/global/GlobalStateConfiguration.java

```diff
@@ -20,21 +22,26 @@
    public static final AttributeDefinition<String> TEMPORARY_LOCATION = AttributeDefinition
          .builder("temporaryLocation", null, String.class)
             .initializer(() -> SecurityActions.getSystemProperty("java.io.tmpdir")).immutable().build();
+   public static final AttributeDefinition<LocalConfigurationManager> LOCAL_CONFIGURATION_MANAGER = AttributeDefinition
+         .builder("localConfigurationManager", null, LocalConfigurationManager.class)
+         .initializer(() -> new EmbeddedLocalConfigurationManager()).immutable().build();
```

Comment:
you can use method reference `EmbeddedLocalConfigurationManager::new`
althoug, using the class name doesn't sound a bad idea either...